test/components/componentTests.ts

```diff
@@ -445,6 +445,20 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
+  it("renderTo() only accepts strings, selections containing svgs, and SVG elements", () => {
+    svg.attr("id", "render-to-test");
+    assert.doesNotThrow(() => c.renderTo("#render-to-test"), Error, "accepts strings that identify svgs");
+    assert.doesNotThrow(() => c.renderTo(svg), Error, "accepts selections that contain svgs");
+    assert.doesNotThrow(() => c.renderTo(<any> svg.node()), Error, "accepts svg elements");
+    let parent = TestMethods.getSVGParent();
+    let div = parent.append("div");
+    // HACKHACK #2614: chai-assert.d.ts has the wrong signature
+    (<any> assert).throws(() => c.renderTo(div), Error, "", "rejects selections that don't contain svgs");
+    (<any> assert).throws(() => c.renderTo("#not-a-element"), Error, "", "rejects strings that don't correspond to DOM elements");
+    (<any> assert).throws(() => c.renderTo(d3.select(null)), Error, "", "rejects empty d3 selections");
```

Comment:
Yea. You can do that very easily by writing some garbage above, see what the test is expecting and just paste it there. It's better than leaving it empty